en/docs/basics_topics/concept_explanation.md

```diff
@@ -1,69 +1,73 @@
+In OneFlow, the basic level also use the concept of Tensor. But there are some difference  about Tensor between OneFlow and pytorch/tensorflow. In order to provide sufficient support for distributed system and parallelism, the Tensor in OneFlow is more complex and have more types and attributes (Such as logical, physical, devices and attributes of distribution). The Tensor unified on logic level, could be divided to different devices. In order to simplify description, OneFlow hides the different types of Tensor, all the things are defined by a higher level concept named Blob.
 
 
 
-Blob in OneFlow have corresponding base class  `BlobDef`. You can print the attribute of  `Blob` when build network. Such as the following script can print  `conv1`, `shape` and `dtype`:
+In OneFlow, Blob has a corresponding base class `BlobDef`. You can print the attributes of  `Blob` when building network. As the following code, we can print  `conv1`'s `shape` and `dtype`:
 
 ```python
 print(conv1.shape, conv1.dtype)
 ```
 
-Blob can only be Placeholder, but also can be the unit to pack the function's parameters.
+Blob can only be Placeholder, but can also be a specific unit that contains values. 
 
 
 
 ### 3.Job Function
 
-In OneFlow, we called training, evaluations, predictions and ratiocination tasks as job function. Job function connect logic of the users and  computing resources management of OneFlow.
+In OneFlow, we call the training, evaluating, predicting and inferential tasks as job function. Job function connects logic of user and  computing resources that managed by  OneFlow.
 
-In OneFlow, all functions need use decorator `@oneflow.global_function` to define as a job functions. By this decorator, we not only can define the model structure, optimize method and other task logic but also can pass the parameters need by job when running the job function. (like: get_train_config()). That can make OneFlow manage our memory and device resources more conveniently.
+In OneFlow, we can use decorator `@oneflow.global_function` to change a function to a job function. By this decorator, we can not only define the type of job function(such as: `type="train"`), but also bind a `FunctionConfig` object to set the configuration of job function. OneFlow can manage our memory and device resources more conveniently.
 
 
 
- **Why use global_function rather than function?**
+ **Why use global_function?**
 
-The beginning of the design of OneFlow is to solve the task of multiple device in distribution training. In this situation, set global_function means make global configuration on devices.
+The beginning of the OneFlow's design is to solve the task of multiple devices in distributed training. In this situation, we set the global configuration by `global_funtion`. 
```

Comment:
The beginning of the OneFlow's design is to solve the task of multiple devices in distributed training.
->
From the begining,  OneFlow want to solve the problem of how to run the jobs on distributed training system with multiple devices.